src/handlers/messageHandler.ts

```diff
@@ -42,7 +43,10 @@ async function botCallBranch(msg: Message<true>) {
 }
 
 async function noBotCallBranch(msg: Message<true>) {
-  if (urlCheck(msg)) urlCheckWarning(msg);
+  if (urlCheck(msg)) {
+    urlCheckWarning(msg);
+    urlCheckLoserboard(msg.guild!.id, msg.author.id);
```

Comment:
This will work as-is, but for the sake of consistency with the rest of the functions, I would go ahead and pass `msg` through as a single parameter, and pick out the parts you need on the other side.